src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -31,7 +31,7 @@ internal class SystemMetadata
         [JsonProperty("_op_gitContributorInformation")]
         public ContributionInfo? ContributionInfo { get; set; }
 
-        public string?[]? GitContributorInformation { get; set; }
+        public string?[]? GitContributors { get; set; }
```

Comment:
```suggestion
        public string[]? GithubContributors { get; set; }
```

Reply:
The type of "Name" property in Contributor.cs is `string?`, thus we have to use `string?[]?` here